subschemas/substrate-chain/src/resolvers/Query/councilMotions.ts

```diff
@@ -107,3 +120,23 @@ function getVotingStatus(
     status,
   };
 }
+
+async function getMotionDetails(
+  motion: DeriveCollectiveProposal,
+  accountService: AccountsService,
+  api: Context['api'],
+  councilMembers: [AccountId, Balance][],
+  bestNumber: BlockNumber,
+): Promise<CouncilMotion | null> {
```

Comment:
Why `| null`?

Reply:
Oh you're right, I refactored but decided to return null on the function above 👍  Will change